addons/terraform-ec/readme.md

```diff
@@ -0,0 +1,3 @@
+# Addon terraform-ec
+
+This addon contains Terraform provider for Elastic Cloud.
```

Comment:
how this addon can be used?

Reply:
This pr will extend terraform addon, support elastic cloud. Now user can play with ec cloud resources in vela. Refer to the doc, first `vela addon enable terraform`, then `vela addon enable terraform-ec`, also need credentials.

Next step, to provision and consume the cloud resources, user/developer would extend the cloud resources, like preparing `ComponentDefinition`. Per this guide: https://kubevela.net/docs/platform-engineers/components/component-terraform

I found that other terraform provider addons' readme file are as simple as this. To keep style consistent, I just commited whole folder with no personal modify. If necessary, we can discuss and add more contents.

Reply:
Let's add more explanation into Readme here

Reply:
You can explain a bit more about what kind of usage users can use.

1. how to enable/disable this addon.
2. what component can they use?
3. what's benefit from this addon/component?

Reply:
Good point!

IMO, 

> how to enable/disable this addon.

User will per this guide https://kubevela.net/docs/reference/addons/terraform, to enable Terraform addon and Terraform Provider addon.

> what component can they use?

Per this guide https://kubevela.net/docs/tutorials/consume-cloud-services, to find supported components.

> what's benefit from this addon/component?

Mainly:

* https://registry.terraform.io/providers/elastic/ec/latest/docs
* https://www.elastic.co/guide/index.html

Also, We can use `vela def doc-gen` to gen standard docs for new `ComponentDefinition`, per [Extend Cloud Resources > Generate documentation](https://kubevela.net/docs/platform-engineers/components/component-terraform#generate-documentation). And submit the docs to [Supported Cloud Resource list](https://kubevela.net/docs/end-user/components/cloud-services/cloud-resources-list) on website.